src/server.js

```diff
@@ -68,6 +92,7 @@ app.put("/presentations/favorite/:id", (req, res) => {
     ? favorites.filter((f) => f !== id)
     : [...favorites, id];
 
+  req.session.favorite_presentations = favorites;
```

Comment:
Modifying session properties will automatically trigger a Set-Cookie header on the response.